database/migrations/2021_08_20_112909_create_countries_place_by_sports.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCountriesPlaceBySports extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('countries_place_by_sports', function (Blueprint $table) {
```

Comment:
@hayaIshida I think you could have left the default Laravel table naming, like `country_sport` for pivot tables.